src/api/dataTypes.js

```diff
@@ -3,6 +3,7 @@ import throttle from 'util/throttle';
 
 export const getDataTypes = (page, params) => new Promise((resolve) => {
   if (params) {
+    // eslint-disable-next-line no-console
     console.info(`calling API /datatypes${params}`);
```

Comment:
We are keeping this as long as we the real API request is not implemented, then we should remove it. I'm shutting down the warning to avoid spamming the console while developing.
Do you think I should avoid shutting down the warning, or maybe remove the log?

Reply:
I'm just afraid we'll forget about removing it if we shut it